src/test/java/de/tum/in/www1/artemis/plagiarism/PlagiarismUtilService.java

```diff
@@ -84,6 +92,15 @@ public Course addCourseWithOneFinishedTextExerciseAndSimilarSubmissions(String u
         return course;
     }
 
+    /**
+     * Creates and saves a ModellingExercise. Also creates and saves StudentParticipations with similar Submissions.
+     *
+     * @param userPrefix             The prefix for the user logins
+     * @param similarSubmissionModel The model that is submitted by each student
```

Comment:
I would change the sentence to "The model that each student submits". Then you don't use passive voice.